subsys/mgmt/mcumgr/Kconfig

```diff
@@ -156,6 +156,17 @@ config IMG_MGMT_UL_CHUNK_SIZE
 	  this size gets allocated on the stack during handling of a image upload
 	  command.
 
+config IMG_MGMT_UPDATABLE_IMAGE_NUMBER
+	int "Number of supported images"
+	default 1
```

Comment:
Can't we have the default set to CONFIG_UPDATEABLE_IMAGE_NUMBER ?

Reply:
@de-nordic zephyr/subsys/dfu/Kconfig defines mentioned property

Reply:
Or maybe even better reuse UPDATEABLE_IMAGE_NUMBER (if that makes sense to you)?